.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
(IDE configuration files like this should not be added to the repository)

Reply:
Noted, this was added by mistake.